packages/sortable/src/components/SortableContext.tsx

```diff
@@ -94,6 +98,7 @@ export function SortableContext({
       useDragOverlay,
       sortedRects: getSortedRects(items, droppableRects),
       strategy,
+      disabled,
```

Comment:
Passing disabled to SortableContext makes it possible to disable all items in the context